### DISCUSSION TITLE
[DONE ✅]  Support Specter Export

### DISCUSSION THREAD

**Discussion by Kukks** [btcpayserver/btcpayserver#2223] [Ideas / Feature Requests]
Specter outputs the following wallet format:
`{ "label": "Hot Wallet p2sh", "blockheight": 0, "descriptor": "sh(wpkh([5c9e228d\/49h\/1h\/0h]tpubDCHRnuvE95JrpEVTUmr36sK3K9ADf3s3aztpXzL8coBeCTE8cHV8PjxS6SjWJM3GfPn798gyEa3dRPgjoUDSuNfuC9xz4PHznwKEk2XL7X1\/0\/*))#yeyuq4c9" }`

Since we already support output descriptors, this should be quite easy.

**Comment by pavlenex** [comment]
Can you explain a bit more about where this would be added? I assume store > `import from`?

**Comment by dennisreimann** [comment]
Yup. I'd like to tackle this 😀

**Reply by dennisreimann** [reply]
Done with #2252.
